src/test/java/com/aquaq/training/test/MainTest.java

```diff
@@ -0,0 +1,176 @@
+package com.aquaq.training.test;
+
+import com.aquaq.training.Main;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+//@ExtendWith(MockitoExtension.class)
+public class MainTest {
+
+    String userString1;
+
+    String userString2;
+
+    private static final PrintStream standardOut = System.out;
+    private static final ByteArrayOutputStream outputStreamCaptor = new ByteArrayOutputStream();
+
+    @BeforeAll
+    public static void setUp() {
+
```

Comment:
move initialisation of outputStreamCaptor to here as we want a new ByteArrayOutputStream for each test